docs/development/workflow/get_devel_version.rst

```diff
@@ -229,6 +229,16 @@ ahead to the more sophisticated method look at :ref:`virtual_envs`).
     + You use Python 3.
```

Comment:
So, is this still true? I thought this works in python3...

Reply:
Am not sure, let me find out...

Reply:
Yes @mhvk, I think this line (and perhaps the entire note) should be removed.

I think nowadays the recommended way to go is to use virtual environment to make the management of software easier and less mistake-prone. So I would suggest removing the parts of the docs that are not consistent with such a practice. I can check the entire document and rewrite/edit if necessary.

Reply:
If this works on python3, I actually think it is useful to keep this, even though the virtual environments is recommended - it is nice for people to be able to start in small steps. I certainly used it a lot when I started to develop myself.

Reply:
Sure, not a problem. It may be good as a legacy reference too.